CHANGELOG.md

```diff
@@ -4,14 +4,38 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## Unreleased
+## 2.4.0 (2021-07-26)
+This release adds new experimental features since the 2.3 release.
+The experimental features in this release are:
+* APIs for chunk manipulation across data nodes in a distributed
+hypertable setup. This includes the ability to add a data node and move
+chunks to the new data node for cluster rebalancing,  
+* time_bucket_ng function, a newer version of `time_bucket`.This function
+supports years, months, days, hours, minutes and seconds. 
+We welcome feedback on the experimental features.
```

Comment:
Instead we should say:
We’re committed to developing these experiments, giving the community a chance to provide early feedback and influence the direction of TimescaleDB’s development. We’ll travel faster with your input! 

Please create your feedback as a GitHub issue (using the experimental-schema label), describe what you found, and tell us the steps or share the code snip to recreate it.

Reply:
added